src/components/ec-table-pagination/ec-table-pagination.spec.js

```diff
@@ -12,6 +12,16 @@ describe('EcTablePagination', () => {
     });
   }
 
+  beforeEach(() => {
+    window.matchMedia = jest.fn().mockImplementation(query => ({
```

Comment:
Is this still needed? If yes, we need to restore it back to the origin in `afterEach` so it doesn't spoil other tests running after this one.

Reply:
It is not needed at all, thanks for noticing!